src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -1101,6 +1202,82 @@ components:
         - generated
         - printed
 
+    AedSearchSummaryResponse:
+      title: AedSearchSummaryResponse
+      description: Response containing the meta data and results of searching for Anonymous Elector document summaries
+      type: object
+      properties:
+        results:
+          type: array
+          description: Array of anonymous elector document search summaries that matches the search criteria
+          items:
+            $ref: '#/components/schemas/AedSearchSummary'
+      required:
+        - results
+
+    AedSearchSummary:
+      title: AedSearchSummary
+      description: An object containing summary information of an each Anonymous Elector document summary
+      type: object
+      properties:
+        gssCode:
+          type: string
+          description: The relevant GSS code for the local authority.
+          example: 'E09000007'
+        sourceReference:
+          type: string
+          pattern: '^[a-fA-F\d]{24}$'
+          description: |
+            A 24 character hex string. 
+            The unique identifier of an Anonymous elector document application.
+          example: 1f0f76a9a66f438b9bb33070
+        applicationReference:
+          type: string
+          description: The reference number of an Anonymous elector document application
+          example: A3JSZC4CRH
```

Comment:
Minor, but the example should be a "V" number, because all the application references always start with "V"

Reply:
Copy pasted from vca-api spec, will change it to start with `V` :)